### ISSUE TITLE
Planning out the 'gettemp' replacement

### ISSUE THREAD

**Issue by astrobokonon** [LowellObservatory/MrFreeze#4]
As things stand right now, I'm facing an existential question.

["Iago"](https://github.com/LowellObservatory/DataServants/blob/master/Iago.py) has been the DCT broker consumer/database storage process for quite some time now.  Iago has been handling the instrument temperatures as reported by LOIS, and contains all the logic for parsing those ugly LOIS log files.

Mr. Freeze is rapidly reaching a deploy-able state; as such, I'd like to get _all_ the temperature/telemetry stuff under his umbrella.

That means doing ONE of the following:

- Removing the temperature parsing ([from parserLOlogs](https://github.com/LowellObservatory/DataServants/blob/9315d79d3b9510ae83deecc420debc2e2cf376e2/dataservants/iago/amqparse.py#L255)) and putting it into Mr. Freeze

- Making something (perhaps [Abu](https://github.com/LowellObservatory/DataServants/blob/master/Abu.py)) repackage the values into a new broker topic

- Something else??

**Comment by astrobokonon** [comment]
In general I would *love* to speed up perserLOlogs; I suspect it's not the most efficient thing I've ever written, especially with all of its string manipulations.

**Comment by astrobokonon** [comment]
I think I'm settled on the first path; that means tearing out the temperature LOIS log parser from Iago, which has already been completed as of [Dataservants@585501e](https://github.com/LowellObservatory/DataServants/commit/585501e55c6d7225f155b2b80ac6bfa1c6e0c222).

**Comment by astrobokonon** [comment]
Just need to fold in queue support when I put in the gettemp support, and then the Mr. Freeze alpha is complete.  It makes sense to do both of those at once because I'll need some topic subscriptions for gettemp support, so I might as well do the queue at the same time since I'll need one for that too.  Both are tied to the custom listener.

**Comment by astrobokonon** [comment]
Ended up making 'queue-' a special subset of the common block and parsed it accordingly.  A little messy, but it seemed like it was the most transparent way of potentially rolling the concept out to other codes

**Comment by astrobokonon** [comment]
Briefly tested the draft implementation in f850fbc3320e1112d21ebb7758204e113c0f6155 and it seemed to work ok.  Giving NIHTS a fuller test overnight but leaving the 'gettemp' stuff turned off for now.

Not _super_ happy with the hardcoding I had to do in the listener, but I made myself feel better about it by saying that it's only really a shim until we replace LOIS.  At least that's what I tell myself.

I'm content to close this now, since it works fine.
